webapp/app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+from models import Call
+from flask.json import jsonify
+
+DEBUG = os.getenv('ENVIRONEMENT') == 'DEV'
+PG_CONFIG = {
+    'user': os.getenv('PG_USERNAME', 'nola311'),
```

Comment:
Prefix this with `NOLA311_` so we can namespace our environment variables to avoid collisions with other apps